ResearchKit/Onboarding/ORKVerificationStepViewController.m

```diff
@@ -0,0 +1,107 @@
+ FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+
+#import "ORKVerificationStepViewController.h"
+#import "ORKStepViewController_Internal.h"
+#import "ORKVerificationStep.h"
+#import "ORKVerificationStepView.h"
+
+
+@implementation ORKVerificationStepViewController {
+    ORKVerificationStepView *_verificationStepView;
+}
+
+- (ORKVerificationStep *)verificationStep {
+    return (ORKVerificationStep *)self.step;
+}
+
+- (void)stepDidChange {
+    [super stepDidChange];
+    
+    if (self.step && [self isViewLoaded]) {
+        _verificationStepView = [[ORKVerificationStepView alloc] initWithFrame:self.view.bounds];
+        _verificationStepView.autoresizingMask = UIViewAutoresizingFlexibleWidth|UIViewAutoresizingFlexibleHeight;
+        _verificationStepView.headerView.captionLabel.text = [self verificationStep].title;
+        _verificationStepView.headerView.instructionLabel.text = [self verificationStep].text;
+        _verificationStepView.emailLabel.text = self.emailAddress;
+        [self.view addSubview:_verificationStepView];
+        
+        [_verificationStepView.resendEmailButton addTarget:self
+                                                   action:@selector(resendEmailButtonHandler:)
+                                         forControlEvents:UIControlEventTouchUpInside];
+        
+        [_verificationStepView.changeEmailButton addTarget:self
+                                                   action:@selector(changeEmailButtonHandler:)
+                                         forControlEvents:UIControlEventTouchUpInside];
+    }
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self stepDidChange];
+}
+
+- (void)setContinueButtonItem:(UIBarButtonItem *)continueButtonItem {
+    [super setContinueButtonItem:continueButtonItem];
+    
+    [continueButtonItem setTarget:self];
+    [continueButtonItem setAction:@selector(continueButtonHandler:)];
+    
+    _verificationStepView.continueSkipContainer.continueButtonItem = continueButtonItem;
+    _verificationStepView.continueSkipContainer.continueEnabled = YES;
+}
+
+- (void)continueButtonHandler:(id)sender{
+    [self continueButtonTapped];
+}
+
+- (void)continueButtonTapped {
+}
+
+- (void)resendEmailButtonHandler:(id)sender {
+    [self resendEmailButtonTapped];
+}
+
+- (void)resendEmailButtonTapped {
+}
+
+- (void)changeEmailButtonHandler:(id)sender {
+    [self changeEmailButtonTapped];
+}
+
+- (void)changeEmailButtonTapped {
+}
+
+- (NSString *)emailAddress {
+    return nil;
```

Comment:
https://github.com/ResearchKit/ResearchKit/blob/7f106898e23dd00057dc08a77591913f851f6bd7/ResearchKit/Charts/ORKGraphChartView.m#L1012-L1016

Should enforce the developer to subclass the required abstract methods (all of them)?